.github/CODEOWNERS

```diff
@@ -89,7 +89,7 @@
 /sdk/communication/azure-communication-phonenumbers/                 @miguhern @whisper6284 @RoyHerrod @danielav7
 
 # PRLabel: %Communication - Rooms
-/sdk/communication/azure-communication-rooms/                        @shirleyqin-msft
+/sdk/communication/azure-communication-rooms/                        @shirleyqin-msft @Azure/acs-rooms-sdk
```

Comment:
do we need to keep shirleyqin-msft as owner?

Reply:
I don't think it hurts to have extra pair of eyes - if they want to remove themselves, that's also fine